doc/doc.html

```diff
@@ -5,6 +5,12 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="docstyle.css">
+  <link
```

Comment:
If this font awesome is for the icons you use in both your documentation website and your components, suggest adding it in your common styles.css of all components. If not, then looks good!

Reply:
Yes sure:)